command-line-options.md

```diff
@@ -0,0 +1,203 @@
+
+ * `-e`
+
+Boot the node in ephemeral mode. Ephemeral nodes are read-only participants in
+the cluster. They do not appear in the cluster topology, and do not accept
+incoming metrics, but may be used to read metric data from other nodes and
+perform intensive computation that would add unreasonable load to the main
+nodes.
+
+## Loader Options
+
+**DEPRECATED**
+
+These options imply foreground operation and perform a specific task, then
+exit. They are only valid in identified mode (`-i`).
+
+ * `-m` 
+
+Merge text reconstitution files.
+
+ * `-H`
+
+Merge histogram reconstitution files. 
+
+These options were used in a previous version of the reconstitute process and
+are no longer strictly required. They may be removed in a future version.
+
+## Maintenance Options
+
+These options imply foreground operation and perform a specific task, then
+exit. They are only valid in identified mode (`-i`).
+
+ * `-r text/metrics`
+
+Repair text inventory.
+
+ * `-r text/changelog`
+
+Repair text datastore.
+
+ * `-r hist/metrics`
+
+Repair histogram inventory.
+
+ * `-r hist/rollups`
+
+Repair histogram rollups.
+
+ * `-j`
+
+Journal-drain mode. Does not start a network listener, so this node will appear
+"down" to its peers, but will send any pending journal data to them.  This is
+useful if you are planning to retire and replace a cluster node, and want to
+ensure that it has sent all outgoing journal data without accepting any new
+input.
+
+## Behavioral Options
+
+These determine optional behavior, and are not required.
+
+ * `-c <file>`
+
+Load configuration from `<file>`. Must be a full path. If not specified, the
+default path is `/opt/circonus/etc/snowth.conf`.
+
+ * `-d`
+
+Activate additional debug logging. Use with caution; can generate a large
+volume of logs.
+
+ * `-D`
+
+Stay in the foreground, rather than daemonizing. If specified once, run as a
+single process with no watchdog. If specified twice, run as a parent/child
+pair, with the parent (watchdog) process in the foreground.
+
+See the [libmtev
+documentation](http://circonus-labs.github.io/libmtev/development/mtev_main.html#foreground)
+for details on foreground operation.
+
+ * `-u <user>`
+
+Drop privileges after start and run as this user.
+
+ * `-g <group>`
+
+Drop privileges after start and run as this group.
+
+ * `-t <path>`
+
+Chroot to `<path>` for operation. Ensure that log file locations may be
+accessed within the chrooted environment.
+
+ * `-l <logname>`
+
+Enable `<logname>`, even if it is disabled in the configuration file. The
+specified log stream must exist.
+
+ * `-L <logname>`
+
+Disable `<logname>`, even if it is enabled in the configuration file. The
+specified log stream must exist.
+
+## Reconstitute Options
+
+These operations are used when [rebuilding a node](/rebuilding-nodes.md).
+
+ * `-B`
+
+Enable reconstitute mode.
+
+ * `-T <topo_hash>`
+
+Reconstitute from this remote/foreign topology. Used when creating a new cluster
```

Comment:
@pamaddox is this line not sufficient to explain the purpose of `-T`?

Reply:
A feel like people are going to wonder _which_ topology to enter there, since there'd be two. I've managed to confuse _myself_ and do it wrong, even though I'm the one that wrote it in the first place. I figure - if I mess it up from time-to-time, other people will too.

Reply:
My intention is to link this to the (as yet unwritten) section on creating a new cluster from an old one.  I believe it will be clear from reading that (and the examples it will contain) what should be put here.  This is really just a quick reference, not meant to fully explain every potential usage.